src/sentry/utils/performance_issues/performance_problem.py

```diff
@@ -43,6 +52,8 @@ def from_dict(cls, data: dict):
             data["parent_span_ids"],
             data["cause_span_ids"],
             data["offender_span_ids"],
+            data["evidence_data"],
+            data["evidence_display"],
```

Comment:
We should do this via `data.get("evidence_data", [])` so that it doesn't break for existing perf problems we're fetching from nodestore.

Reply:
ah 👍 that might fix these 10000 failing tests

Reply:
updated!